src/runtime_src/core/pcie/driver/linux/xocl/xocl_subdev.c

```diff
@@ -1685,7 +1689,11 @@ int xocl_subdev_create_vsec_devs(xdev_handle_t xdev)
 			    (subdev_info.priv_data))->flash_type,
 			    FLASH_TYPE_QSPIPS, strlen(FLASH_TYPE_QSPIPS));
 			/* default is FLASH_TYPE_SPI, thus pass through. */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 4, 0)
```

Comment:
Same question here.

Reply:
It depends on the very first version that linux kernel start supporting the new APIs, we want the exact version  as a split line.